libnmstate/netapplier.py

```diff
@@ -150,6 +175,14 @@ def _apply_ifaces_state(
                 _disable_ipv6(desired_state)
             if verify_change:
                 _verify_change(desired_state)
+            if vlan_ifaces_state:
+                _apply_edited_ifaces_without_checkpoint(
+                    state.State({schema.Interface.KEY: vlan_ifaces_state}),
+                    verify_change=False,
```

Comment:
Always disable verification should not be needed, if at all it might be ok for the first apply as long as the changes are verified after the second apply to ensure that the final outcome state is as expected.

Reply:
Yes, as I said above we are not using that argument anymore. Thanks!